File: backend/src/main/kotlin/com/refined/stone/reaction/Reaction.java
//package com.refined.stone.reaction;
//
//import com.refined.stone.account.Account;
//
//import javax.persistence.*;
//
//@Entity
//public class Reaction {
//    @Id
//    @GeneratedValue(strategy = GenerationType.IDENTITY)
//    private Long Id;
//
//    @Column
//    private Long avgReactionTime;
//
//    //many reaction result to one account
//    @ManyToOne(fetch = FetchType.LAZY)
//    @JoinColumn(name = "account_id")
//    Account account;
//}

File: backend/src/main/kotlin/com/refined/stone/reaction/Reaction.kt
package com.refined.stone.reaction

import com.refined.stone.account.Account
import javax.persistence.*

@Entity
class Reaction(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private val id: Long? = null,

    @Column
    private val avgReactionTime: Long=0,

    //many reaction result to one account
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "account_id")
    var account: Account? = null
) {



}